soepy/python/shared/shared_auxiliary.py

```diff
@@ -112,20 +111,19 @@ def calculate_consumption_utilities(attr_dict, period_wages):
         print(consumption_utilities)
         sys.exit("Error message")
```

Comment:
If you would like to check this, then please use https://docs.scipy.org/doc/numpy/reference/routines.testing.html instead. We do want to raise proper exceptions for all errors.

Reply:
I removed the entire chunk of code. It was in there as it helped me and Sebastian detect a bad parameter input. We should probably implement handling of such errors with appropriate Error Messages in a separate issue.